app/views/student_teams/view.html.erb

```diff
@@ -90,6 +116,51 @@
   </table>
 <% end %>
 
+<% if @student.team != nil and Team.find(@student.team.id).pair_programming_request!=0 %>
```

Comment:
when using if not, use unless instead. use && instead of and. since we are using unless we need to apply demorgans law to get this result
```suggestion
<% unless @student.team.nil? || Team.find(@student.team.id).pair_programming_request.zero? %>
```